ui/client/components/graph/node-modal/node/NodeDetails.tsx

```diff
@@ -105,6 +105,11 @@ export function NodeDetails(props: NodeDetailsProps): JSX.Element {
     return {HeaderTitle}
   }, [node])
 
+  //no process? no nodes? no window contents! no errors for whole tree!
```

Comment:
what about empty scenario?

Reply:
nodeDetails on empty scenario?

Reply:
I see... properties... but it's not this case. empty scenario has empty array in nodes.